e2e/pages/enterAllocationProposal/enterAllocationProposal.js

```diff
@@ -0,0 +1,29 @@
+const I = actor();
+
+module.exports = {
+
+  fields: {
+    allocationProposal: '#allocationProposal_proposal',
+    proposalReason: '#allocationProposal_proposalReason',
+  },
+
+  links: {
+    presidentsGuidance: 'President\'s Guidance',
+    schedule: 'Schedule',
+  },
+
+  selectAllocationProposal(proposal) {
+    I.waitForElement(this.fields.allocationProposal);
+    I.selectOption(this.fields.allocationProposal, proposal);
+  },
+
+  enterProposalReason(reason) {
+    I.fillField(this.fields.proposalReason, reason);
+  },
+
+  clickDocumentLink(document) {
```

Comment:
That appears to be not used.

Reply:
You're right, was replaced with the function from the steps_file